cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiTest.java

```diff
@@ -290,4 +290,28 @@ public void testChangeLoggers(VertxTestContext context) throws InterruptedExcept
                             async.flag();
                         }))));
     }
-}
+
+    @IsolatedTest
+    public void testHierarchy() {
```

Comment:
This is a test of getLevel/getEffectiveLevel, it doesn't really test the overall functionality. Given we've burned our fingers of this once, I think we need that higher level test.